cst438-project-airline/src/main/java/cst438/domain/UserRepository.java

```diff
@@ -1,7 +1,10 @@
 package cst438.domain;
 
+import java.util.ArrayList;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
```

Comment:
I keep seeing this. What does this do?

Reply:
It turns out that this line is the import needed in order to have the @Repository annotation.